lib/models/event.dart

```diff
@@ -7,7 +8,10 @@ part 'event.g.dart';
 
 @freezed
 class Event with _$Event {
+  const Event._();
+
   const factory Event({
+    @JsonKey(includeFromJson: false, includeToJson: false) String? id,
```

Comment:
hmm to juz jest ustawione w mojej prce

Reply:
No było ale twojej prki wtedy nie było, ustawie żeby było zgodne ze sobą.